Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -261,7 +261,7 @@ checkLibrary pkg lib =
   , checkVersion [1,25] (not (null (signatures lib))) $
       PackageDistInexcusable $
            "To use the 'signatures' field the package needs to specify "
-        ++ "at least 'cabal-version: >= 1.25'."
+        ++ "at least 'cabal-version: >= 2.0'."
```

Comment:
Can we please normalise to recommending `cabal-version: 2.0` w/o the `>=`? :-)

Reply:
Good point. Will do and merge after that.